Src/Extern/HYPRE/AMReX_HABEC_3D.F90

```diff
@@ -11,6 +11,7 @@ module amrex_habec_module
 
   use amrex_fort_module, only : rt => amrex_real
   implicit none
+  integer(kind=8), parameter :: HYPRE_PARCSR = 5555
```

Comment:
This integer parameter is not used anywhere. Do we really need it?

Reply:
Good catch. I'll remove it.